modules/gapi/test/common/gapi_imgproc_tests_inl.hpp

```diff
@@ -380,6 +399,152 @@ TEST_P(CannyTest, AccuracyTest)
     }
 }
 
+TEST_P(OptFlowLKTest, AccuracyTest)
```

Comment:
This test duplicates 50% of code of the performance test. Can common stuff be somehow shared between?
May be initialization function can made as a test parameter and be shared among the instantiations and so on?